### DISCUSSION TITLE
Status of OIDC connector as beta

### DISCUSSION THREAD

**Discussion by scor** [dexidp/dex#2149] [Q&A]
As I was reviewing the [list of connectors](https://github.com/dexidp/dex#connectors) Dex offers, I noted that the OpenID Connect connector was marked as "beta", which I found intriguing for "a federated OpenID Connect provider" like Dex. I understand there is 2 levels of OpenID Connect at play: 1) the protocol that clients use to talk to Dex and 2) the protocol that Dex uses through the connector to talk to the actual IdP. However I'm confused why the second would be considered as "beta" if the first is considered "stable" (as I would hope it to be the case). Could someone explain if beta is accurate here and what this mean?

**Comment by nabokihms** [comment]
Good question @scor.
In the README.md under the table we have a description of what we meant to be stable, alpha and beta.
```
Stable, beta, and alpha are defined as:

Stable: well tested, in active use, and will not change in backward incompatible ways.
Beta: tested and unlikely to change in backward incompatible ways.
Alpha: may be untested by core maintainers and is subject to change in backward incompatible ways.
```

**Reply by sagikazarmark** [reply]
It's probably a matter of updating the status. We should probably go through all connectors and check whether the status is still correct. I consider the OIDC one stable.
